paper-server/patches/sources/net/minecraft/world/entity/projectile/windcharge/BreezeWindCharge.java.patch

```diff
@@ -0,0 +1,26 @@
+--- a/net/minecraft/world/entity/projectile/windcharge/BreezeWindCharge.java
++++ b/net/minecraft/world/entity/projectile/windcharge/BreezeWindCharge.java
+@@ -20,6 +_,12 @@
+ 
+     @Override
+     public void explode(Vec3 pos) {
++        // Paper start - Breeze Wind Charge event
++        org.bukkit.event.entity.ExplosionPrimeEvent event = org.bukkit.craftbukkit.event.CraftEventFactory.callExplosionPrimeEvent(this, 3.0F, false);
```

Comment:
You can use the constant RADIUS, same for the other class

Reply:
man I really need to read. I was looking for something like that